test/parallel/test-tls-invalid-session.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+const common = require('../common');
```

Comment:
This file is basically entirely copied from `test-tls-session-cache.js` with tweaks, so it's probably doing more than it actually needs to.